lib/hammer_cli_foreman/compute_profile.rb

```diff
@@ -0,0 +1,80 @@
+require 'hammer_cli_foreman/image'
+
+module HammerCLIForeman
+
+  class ComputeProfile < HammerCLIForeman::Command
+    resource :compute_profiles
+
+    class ListCommand < HammerCLIForeman::ListCommand
+
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+      end
+
+      build_options
+    end
+
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+
+      output ListCommand.output_definition do
+        HammerCLIForeman::References.taxonomies(self)
+        HammerCLIForeman::References.timestamps(self)
+
+        collection :compute_attributes, _("Compute attributes") do
+          field :id, _('Id')
+          field :name, _('Name')
+          field :compute_resource_id, _('Compute Resource Id')
+          field :compute_resource_name, _('Compute resource name')
```

Comment:
API also returns `compute_profile_{id,name}` do you think I should add them here too? Or `Name`, `Id` are sufficient?